From 38a74b13abad8a2c3e8e330121ef73c155ebf16c Mon Sep 17 00:00:00 2001
From: Oliver Specht <oliver.specht@cib.de>
Date: Wed, 7 Oct 2015 16:16:20 +0200
Subject: [PATCH 2/2] tdf#94804: prefix to save cross reference bookmark fixed

bookmarks as references to numbered paragraphs are now saved
using the prefix '__RefNumPara__' as the import already expects

Change-Id: I6f7c41e95fa9f936a7eb21d4cb7461d4575c492a
Reviewed-on: https://gerrit.libreoffice.org/19229
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Michael Stahl <mstahl@redhat.com>
(cherry picked from commit 85eb4d9514858c71fb22c5ed93d651ea24bc6c05)
Reviewed-on: https://gerrit.libreoffice.org/19338
Tested-by: Michael Stahl <mstahl@redhat.com>
(cherry picked from commit bdc5aab39be149b2fc7d37520414ae251341bc14)
---
 sw/source/core/crsr/crossrefbookmark.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sw/source/core/crsr/crossrefbookmark.cxx b/sw/source/core/crsr/crossrefbookmark.cxx
index 22438a8..9ee1a38 100644
--- a/sw/source/core/crsr/crossrefbookmark.cxx
+++ b/sw/source/core/crsr/crossrefbookmark.cxx
@@ -78,7 +78,7 @@ namespace sw { namespace mark
         const KeyCode& rCode,
         const OUString& rName,
         const OUString& rShortName)
-        : CrossRefBookmark(rPaM, rCode, rName, rShortName, IDocumentMarkAccess::GetCrossRefHeadingBookmarkNamePrefix())
+        : CrossRefBookmark(rPaM, rCode, rName, rShortName, CrossRefNumItemBookmark_NamePrefix)
     { }
 
     bool CrossRefNumItemBookmark::IsLegalName(const OUString& rName)
-- 
2.4.3

